.github/dependabot.yml

```diff
@@ -30,11 +32,13 @@ updates:
     schedule:
       interval: "weekly"
     ignore:
+    # Managed by cisagov/skeleton-packer
       - dependency-name: "hashicorp/aws"
 
   - package-ecosystem: "terraform"
     directory: "/terraform-post-packer"
     schedule:
       interval: "weekly"
     ignore:
+    # Managed by cisagov/skeleton-packer
       - dependency-name: "hashicorp/aws"
```

Comment:
```suggestion
    # Managed by cisagov/skeleton-packer
    ignore:
      - dependency-name: "hashicorp/aws"
```

Reply:
@jsf9k I followed suit from the `skeleton-generic` code block:

```  
ignore:
      # Managed by cisagov/skeleton-generic
```

Shouldn't we stay consistent in whether we're putting the comment before or after the ignore line?

Reply:
OK, but then this PR would be inconsistent with cisagov/samba-packer#34.  I assumed that was the model since it kept the comments in [the same place as in cisagov/skeleton-packer](https://github.com/cisagov/skeleton-packer/blob/develop/.github/dependabot.yml#L34-L44).

Reply:
Interesting - I didn't notice the skeleton itself was inconsistent (and you're right, so was I).  Unless there's opposition, I'll match the skeleton and it can be fixed there in a future PR if desired

Reply:
Yes, that's the thing to do.

Reply:
Fixed in 8a5f82c